build/patch/pre-autogen/libevent/common/000.patch

```diff
@@ -54,11 +54,21 @@ diff --git a/configure.ac b/configure.ac
 index 51725d6..86da55e 100644
 --- a/configure.ac
 +++ b/configure.ac
-@@ -290,7 +290,7 @@ AC_HEADER_TIME
+@@ -112,7 +112,6 @@ dnl Checks for libraries.
+ AC_SEARCH_LIBS([inet_ntoa], [nsl])
+ AC_SEARCH_LIBS([socket], [socket])
+ AC_SEARCH_LIBS([inet_aton], [resolv])
+-AC_SEARCH_LIBS([clock_gettime], [rt])
+ AC_SEARCH_LIBS([sendfile], [sendfile])
+ 
+ dnl - check if the macro WIN32 is defined on this compiler.
+@@ -289,8 +288,8 @@ AC_C_INLINE
+ AC_HEADER_TIME
  
  dnl Checks for library functions.
- AC_CHECK_FUNCS([gettimeofday vasprintf fcntl clock_gettime strtok_r strsep])
+-AC_CHECK_FUNCS([gettimeofday vasprintf fcntl clock_gettime strtok_r strsep])
 -AC_CHECK_FUNCS([getnameinfo strlcpy inet_ntop inet_pton signal sigaction strtoll inet_aton pipe eventfd sendfile mmap splice arc4random arc4random_buf issetugid geteuid getegid getprotobynumber setenv unsetenv putenv sysctl])
++AC_CHECK_FUNCS([gettimeofday vasprintf fcntl strtok_r strsep])
 +AC_CHECK_FUNCS([getnameinfo strlcpy inet_ntop inet_pton signal sigaction strtoll inet_aton pipe eventfd sendfile mmap splice arc4random arc4random_buf arc4random_addrandom issetugid geteuid getegid getprotobynumber setenv unsetenv putenv sysctl])
  AC_CHECK_FUNCS([umask])
```

Comment:
In this patch we get rid of `clock_gettime()` which is mainly used by libevent to implement precise timing but which is also something that's is only available on iOS >= 10.0. A better patch could be implemented like I already did for getentropy() for libressl, but for the purpose of blessing v0.4.0-beta this one would do.